### PR TITLE
Revert wrong merge from master->stable9.1

### PR DESCRIPTION
**Pull request by PVince81**
Reverts the totality of this merge commit 7c4108ee4b1b3ea1e236d3343ffe08c2589c01d7

I left out the mailmap change as it's fine to keep it.

@DeepDiver1975 @jvillafanez as discussed

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@cdamken there is some beer you have to pass over to @PVince81 and @jvillafanez

File: apps/user_ldap/lib/User/Manager.php [hunk 1]
```diff
@@ -63,13 +62,14 @@ class Manager {
...
+	 * 	['byUid']	\OCA\User_LDAP\User\User[]
+	 * @var array $users
 	 */
-	protected $usersByDN;
-	/**
-	 * @var CappedMemoryCache $usersByUid
-	 */
-	protected $usersByUid;
+	protected $users = array(
+		'byDN'  => array(),
```
Comment (jvillafanez) [inline apps/user_ldap/lib/User/Manager.php]: These changes belong to an already merged PR in stable9.1 (https://github.com/owncloud/core/pull/25326) :worried:

Comment (PVince81) [inline apps/user_ldap/lib/User/Manager.php]: Argh... then it sounds like I'll have to find the wrong commits manually

**Comment by PVince81** [comment]
I'll check all commits after 9.1.0RC1 and check which are supposed to be there.

@DeepDiver1975 there are many transifex ones, I'll leave them and then later you can rerun the script for 9.1 ?

**Pull request closed without merging**